front/src/components/InsertPositionsPanel/InsertPositionsPanel.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import "./InsertPositionsPanel.css";
+import PositionRow from "./PositionRow/PositionRow";
+import mockData from "./positionsData.json";
+import Button from "./../UI/Button/Button";
+
+function InsertPositions() {
+    let [boundaries, setBoundaries] = useState({ min: 0, max: 10 });
+
+    let renderRows = mockData.map((el) => {
+        return (
+            <PositionRow
+                key={el.licence}
+                min={boundaries.min}
+                max={boundaries.max}
+                {...el}
+            />
+        );
+    });
+
+    const handleMinChange = (event) => {
+        setBoundaries({
+            min: event.target.value,
+            max: boundaries.max,
+        });
+    };
+
+    const handleMaxChange = (event) => {
+        setBoundaries({
+            min: boundaries.min,
+            max: event.target.value,
+        });
+    };
+
+    return (
+        <div className="InsertPositionsPanel">
+            <div className="container">
```

Comment:
Replace "container" with something more meaningful (i.e. "InsertPositionsPanelContainer" would fit better here)

Reply:
are you sure? we have a global css class container, its used also in other components and its name is "container" - in order to name it InsertPositionsPanelContainer I would need to copy container class rules from another file and duplicate it with another name - I wont do that now, if you think I should just let me know and I will make that change

Reply:
In that case you can leave it as it is